mail-integration-webapp/src/main/webapp/vue-app/common/main.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2022 eXo Platform SAS.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+import MailIntegrationNotificationAlert from './MailIntegrationNotificationAlert.vue';
+
+const components = {
+  'mail-integration-notification-alert': MailIntegrationNotificationAlert,
```

Comment:
```suggestion
  'mail-integration-alert': MailIntegrationAlert,
```
The name is confusing with the mail integration notification component, this component is a common component it should be named regardless the component in which it will be used